roles/pulp_common/tasks/install_pip.yml

```diff
@@ -132,6 +132,15 @@
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
       when: prereq_pip_packages | length > 0
 
+    - name: Install patched versions of dependencies (FIPS only)
+      pip:
+        name:
+          - git+https://github.com/mdellweg/django.git@fips
+          - git+https://github.com/mdellweg/drf-spectacular.git@fips
```

Comment:
I think we can pull this out actually https://github.com/tfranzel/drf-spectacular/pull/148#issuecomment-691170667

Reply:
Also @mdellweg I don't think you'll need to have create an rpm patch at all because of ^. One ^ releases instead version bump the requirements string for pulpcore's inclusion of drf-spectacular.

Reply:
Didn't use the PR head because it was based on master instead of the latest release. But with a new upstream release, that is even better.

Reply:
Yeyah! `0.9.13` is released! https://github.com/tfranzel/drf-spectacular/pull/148#issuecomment-691644373